### PR TITLE
ArchivesSpace: use display_title if present

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -264,9 +264,14 @@
                on-node-toggle="on_toggle(node, expanded)"
                expanded-nodes="expanded_nodes">
           <!-- ArchivesSpace record -->
-          <span ng-if="node.type === 'resource' || node.type === 'resource_component'" tree-droppable on-drop="drop" file-type="record">
+          <span ng-if="node.type === 'resource'" tree-droppable on-drop="drop" file-type="record">
             {{ node.title }} <div ng-if="node.identifier">({{ node.identifier }})</div>
           </span>
+          <span ng-if="node.type === 'resource_component'" tree-droppable on-drop="drop" file-type="record">
+            <div ng-if="node.display_title">{{ node.display_title }}</div>
+            <div ng-if="!node.display_title">{{ node.title }}</div>
```
Comment (Hwesta) [inline app/index.html:272]: This has a `<div>` but the resource one doesn't - is that a problem?

**Comment by Hwesta** [comment]
:+1:

File: app/index.html (hunk 1 above)
Comment (mistydemeo) [inline app/index.html:272]: I don't think we're styling it differently; it has a `<div>` just so I can tack on the if statement.

Comment (Hwesta) [inline app/index.html:272]: Ok, that's fine

**Pull request merged**